robot/impl/resource_manager_test.go

```diff
@@ -1974,22 +1981,24 @@ func (rr *dummyRobot) Version(ctx context.Context) (robot.VersionResponse, error
 
 // managerForDummyRobot integrates all parts from a given robot except for its remotes.
 // It also close itself when the test and all subtests complete.
-func managerForDummyRobot(t *testing.T, robot robot.Robot) *resourceManager {
+func managerForDummyRobot(t *testing.T, rbt robot.Robot) *resourceManager {
```

Comment:
flyby: trying to avoid using `robot` since it's also a package name - this kind of collision is totally allowed in golang but leads to some confusing autocomplete behavior for me.